### DISCUSSION TITLE
challenge contracts

### DISCUSSION THREAD

**Discussion by Derastephh** [Cyfrin/foundry-full-course-cu#3140] [Q&A]
I am just starting the solidity course, however i don't want to just code along and while you mentioned us trying the challenge contracts, i can't seem to find the challenge contracts. Please help

**Comment by EngrPips** [comment]
Hello @Derastephh, Please read this discussion https://github.com/Cyfrin/foundry-full-course-cu/discussions/2018. It will help you with finding the challenge contract.
